Cleanup memory passes tests.

hsharma35 opened the pull request:
Differential Revision: D68446633

mcremon-meta reviewed (approved):
LGTM, thanks for the cleanup!

mcremon-meta commented:
I guess linter is unhappy

hsharma35 committed:
Cleanup memory passes tests. (#7788)

Summary:
Add verifiers for memory allocation.


Reviewed By: zonglinpeng, mcremon-meta

Differential Revision: D68446633
